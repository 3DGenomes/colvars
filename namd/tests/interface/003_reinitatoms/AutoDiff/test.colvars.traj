# step       
           0 
# step         rgyr                  ft_rgyr               fa_rgyr               
           0    6.87962472939909e+00  0.00000000000000e+00 -8.79624729399095e-01 
           1    6.87884690576167e+00  9.41011309459353e+01 -8.78846905761671e-01 
           2    6.87828544620917e+00  9.04041844059968e+01 -8.78285446209173e-01 
           3    6.87791917633219e+00  8.14481165806775e+01 -8.77919176332188e-01 
           4    6.87771596366921e+00  6.78179862575362e+01 -8.77715963669207e-01 
           5    6.87763583984369e+00  5.08218206455769e+01 -8.77635839843695e-01 
           6    6.87763510984747e+00  3.21944582169370e+01 -8.77635109847473e-01 
           7    6.87767070515395e+00  1.37842105362566e+01 -8.77670705153955e-01 
           8    6.87770424862566e+00 -2.67218755378558e+00 -8.77704248625659e-01 
           9    6.87770538491566e+00 -1.57406166805562e+01 -8.77705384915658e-01 
          10    6.87765396033178e+00 -2.44703048777621e+01 -8.77653960331782e-01 
          11    6.87754086425991e+00 -2.84781933933865e+01 -8.77540864259909e-01 
          12    6.87736785259684e+00 -2.78153473089150e+01 -8.77367852596839e-01 
          13    6.87714714763978e+00 -2.26302115807328e+01 -8.77147147639783e-01 
          14    6.87690017590140e+00 -1.34489740538550e+01 -8.76900175901402e-01 
          15    6.87665557061129e+00 -1.12489580904910e+00 -8.76655570611288e-01 
          16    6.87644626424549e+00  1.30850351689100e+01 -8.76446264245493e-01 
          17    6.87630623826635e+00  2.77657826415640e+01 -8.76306238266348e-01 
          18    6.87626719198701e+00  4.14611442641241e+01 -8.76267191987005e-01 
          19    6.87635569101452e+00  5.29205207769189e+01 -8.76355691014516e-01 
          20    6.87659105493692e+00  6.12164358381422e+01 -8.76591054936916e-01 
# step         rgyr                  ft_rgyr               fa_rgyr               
          20    6.79302548471894e+00                  -nan -7.93025484718935e-01 
          21    6.79123478804224e+00  1.47884739101504e+01 -7.91234788042243e-01 
          22    6.78960135015216e+00  6.31388772773049e+01 -7.89601350152160e-01 
          23    6.78822556441034e+00  1.06473629117398e+02 -7.88225564410344e-01 
          24    6.78718836728478e+00  1.41615537771249e+02 -7.87188367284780e-01 
          25    6.78654524635830e+00  1.65951779056353e+02 -7.86545246358304e-01 
          26    6.78632262212756e+00  1.77868414642072e+02 -7.86322622127558e-01 
          27    6.78651664924570e+00  1.76780555403036e+02 -7.86516649245702e-01 
          28    6.78709447322938e+00  1.63158774650254e+02 -7.87094473229380e-01 
          29    6.78799770245370e+00  1.38435193019198e+02 -7.87997702453698e-01 
          30    6.78914772242091e+00  1.04850106085371e+02 -7.89147722420910e-01 
          31    6.79045239785112e+00  6.52615921470754e+01 -7.90452397851117e-01 
          32    6.79181359951293e+00  2.29066423056178e+01 -7.91813599512934e-01 
          33    6.79313491716379e+00 -1.88723942265561e+01 -7.93134917163787e-01 
          34    6.79432888384693e+00 -5.69200801246805e+01 -7.94328883846926e-01 
          35    6.79532313655605e+00 -8.85212915265794e+01 -7.95323136556052e-01 
          36    6.79606495915343e+00 -1.11649846149003e+02 -7.96064959153426e-01 
          37    6.79652397458270e+00 -1.25081475905038e+02 -7.96523974582701e-01 
          38    6.79669290881434e+00 -1.28439758480306e+02 -7.96692908814337e-01 
          39    6.79658645171890e+00 -1.22194713349711e+02 -7.96586451718904e-01 
          40    6.79623847172641e+00 -1.07555765691685e+02 -7.96238471726408e-01 
