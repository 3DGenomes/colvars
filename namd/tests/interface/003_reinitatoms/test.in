colvar {
    name rgyr
    outputAppliedForce on
    outputTotalForce on
    gyration {
        atoms {
            psfSegID HA
            atomNameResidueRange CA 1-15
        }
    }
} 

harmonic {
    colvars rgyr
    forceConstant 1.0
    centers 6.0
}
