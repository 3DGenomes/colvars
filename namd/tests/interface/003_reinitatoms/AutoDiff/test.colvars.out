colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  http://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is available.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Writing to colvar trajectory file "test.colvars.traj".
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # indexFile = "" [default]
colvars: # smp = on [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 100 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: Warning: no collective variables defined.
colvars: Collective variables initialized, 0 in total.
colvars: No collective variables biases were defined.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # smp = on [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1 [default]
colvars: # colvarsRestartFrequency = 100 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rgyr"
colvars:   Initializing a new "gyration" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { HA }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 15 atoms initialized: total mass = 180.165, total charge = 1.05.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rgyr }
colvars:   # outputEnergy = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 6 }
colvars:   # targetCenters = { 6 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 1
colvars:   # targetForceConstant = -1 [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (15 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # indexFile = "" [default]
colvars: # smp = on [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 100 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: Warning: no collective variables defined.
colvars: Collective variables initialized, 0 in total.
colvars: No collective variables biases were defined.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # smp = on [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1 [default]
colvars: # colvarsRestartFrequency = 100 [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "rgyr"
colvars:   Initializing a new "gyration" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { HA }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "atoms" defined, 15 atoms initialized: total mass = 180.165, total charge = 1.05.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = on
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rgyr }
colvars:   # outputEnergy = off [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 6 }
colvars:   # targetCenters = { 6 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 1
colvars:   # targetForceConstant = -1 [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (15 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
